plugins/node/opentelemetry-instrumentation-pg/test/pg.test.ts

```diff
@@ -563,5 +563,113 @@ describe('pg', () => {
         client.query('SELECT NOW()').then(queryHandler);
       });
     });
+
+    it('should call postQueryHook with query text if set', async () => {
```

Comment:
You can wrap this section with a `describe` to group all these tests under `postQueryHook`.
This will also be consistent with the existing tests for `responseHook` [see here](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/81b3190af64bda14f87c5b0cbd6172bafda26408/plugins/node/opentelemetry-instrumentation-pg/test/pg.test.ts#L397)